curriculum/challenges/english/25-front-end-development/lab-business-card/6690e10ebe2181212abc9652.md

```diff
@@ -0,0 +1,419 @@
+---
+id: 6690e10ebe2181212abc9652
+title: Design a Business Card
+challengeType: 14
+dashedName: lab-business-card
+demoType: onClick
+---
+
+# --description--
+
+**Objective:** Fulfill the user stories below and get all the tests to pass to complete the lab. 
+
+**User Stories:**
+
+1. You should link the style sheet `styles.css` using the `link` tag in the `head` section of the HTML file.
+1. Your page background should be `rosybrown`. The overall font should be `Arial`, with a fallback of `sans-serif`.   
+1. All of the `p` elements should have a top and bottom margin of `5px`.
+1. All links on the page should have a default of no underline. When you hover over the links, there should be an underline.
```

Comment:
```suggestion
1. By default, all links on the page should not be underlined. When you hover over the links, they should be underlined.
```
The first sentence implied (to me) that "no underline" was a property that the links could have.